src/main/java/seedu/duke/command/CommandListClientsWithTags.java

```diff
@@ -34,43 +41,51 @@ public void execute(Ui ui, Storage storage, PropertyList propertyList, ClientLis
             displayClientShort(clientList, ui);
             break;
         default:
+            //no errors are possible at this stage. Any potential errors are handled earlier
```

Comment:
This is in relation to line 32 and line 35. I am not able to comment on that line as it was not in the current code change.

For the method `displayClientNames` and `displayClientEmails`, since it's displaying 1 name and 1 email, I think there's a grammatical error. 

Sorry for missing it out.